source/templates/_auto_code_signing.slim

```diff
@@ -21,7 +21,7 @@
 						span After running the script,
 						strong< upload all exported files
 						span< below:
-					= partial "templates/tab-code_signing-certificates"
+					== include_slim "_tab-code_signing-certificates"
```

Comment:
single `=` supposed to html encode included templates according to the doc, no idea how this could work so far